testbed/testbed/go.mod

```diff
@@ -32,6 +33,8 @@ require (
 	github.com/jbenet/goprocess v0.1.4
 	github.com/libp2p/go-libp2p v0.11.0
 	github.com/libp2p/go-libp2p-core v0.6.1
+	github.com/libp2p/go-libp2p-gostream v0.2.1
+	github.com/libp2p/go-libp2p-http v0.1.6-0.20210310045043-5508c68db693
```

Comment:
This commit configures the HTTP client to use the same buffer size as Graphsync (4 mb) while reading from the sockets.